profitbricks-rest/src/main/java/org/apache/jclouds/profitbricks/rest/domain/Location.java

```diff
@@ -49,9 +46,16 @@ public static Location fromValue(String v) {
    }
 
    public static Location fromId(String id) {
-      for (Location location : values())
-         if (location.id.equals(id))
+      for (Location location : values()) {
+         if (location.id.equals(id)) {
             return location;
+         }
+      }
       return UNRECOGNIZED;
    }
+
+   @Override
+   public String toString() {
+      return id;
```

Comment:
Make this more descriptive, something like:
```
   return "Location[id=" + id + ", description=" + description + "]";
```

..or don't override it at all, and use the actual `getId()` method, to print out the ID.